config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -3447,7 +3447,7 @@ presubmits:
         - '--node-test-args=--feature-gates="ResourceHealthStatus=true" --service-feature-gates="ResourceHealthStatus=true" --runtime-config=api/alpha=true,api/beta=true --container-runtime-endpoint=unix:///run/containerd/containerd.sock --container-runtime-process-name=/usr/bin/containerd --container-runtime-pid-file= --kubelet-flags="--cgroup-driver=systemd --cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/containerd.service" --extra-log="{\"name\": \"containerd.log\", \"journalctl\": [\"-u\", \"containerd\"]}"'
         - --node-tests=true
         - --provider=gce
-        - '--test_args=--timeout=1h --label-filter="FeatureGate: containsAny ResourceHealthStatus && FeatureGate: isSubsetOf ResourceHealthStatus && !Flaky && !Slow"'
```

Comment:
The test is labeled with `NodeFeatureGate: ResourceHealthStatus`, not `FeatureGate: ResourceHealthStatus`. I suspect you need to use `NodeFeatureGate` here.

I don't know why SIG Node uses `NodeFeatureGate`. There is https://github.com/kubernetes/enhancements/tree/master/keps/sig-testing/3041-node-conformance-and-features which outlines some plan, but I have no idea how well that matches the current jobs.

Reply:
```console
$ go test -v ./test/e2e_node -args --list-tests | grep ResourceHealthStatus
    device_plugin_failures_pod_status_test.go:183: [sig-node] Device Plugin Failures Pod Status [NodeFeature:ResourceHealthStatus] will report a Device Status for the failed pod in the pod status
    device_plugin_failures_pod_status_test.go:125: [sig-node] Device Plugin Failures Pod Status [NodeFeature:ResourceHealthStatus] will report a Healthy and then Unhealthy single device in the pod status
```

Reply:
That is a really handy command.

Is it really necessary for the isSubSetOf and the !SKIP && !FLAKY?

Reply:
Pushed a fix.

Reply:
`NodeFeature: isSubsetOf ResourceHealthStatus` ensures that a test which has some other `NodeFeature` besides `ResourceHealthStatus` doesn't run. This is irrelevant now (there is no such tests).

As with the question whether to skip `Flaky` and `Slow` tests the decision is up to you (= SIG Node).